src/Shell/GettextShell.php

```diff
@@ -268,7 +270,7 @@ private function analyzePoFile($filename): void
         if ($numItems > 0) {
             $percent = number_format($translated * 100. / $numItems, 1);
         }
-        $this->out(sprintf('Translated %s of items - %s %', $translated, $numItems, $percent));
+        $this->out(sprintf('Translated %s of %s items - %s %', $translated, $numItems, $percent));
```

Comment:
```suggestion
        $this->out(sprintf('Translated %s of %s items - %s %%', $translated, $numItems, $percent));
```

In `sprintf` a `%` is required to escape a `%` symbol